tests/scripts/test_psa_constant_names.py

```diff
@@ -348,13 +359,9 @@ def run_tests(options, inputs):
     """
     count = 0
     errors = []
-    for type_word, names in [('status', inputs.statuses),
-                             ('algorithm', inputs.algorithms),
-                             ('ecc_curve', inputs.ecc_curves),
-                             ('dh_group', inputs.dh_groups),
-                             ('key_type', inputs.key_types),
-                             ('key_usage', inputs.key_usage_flags)]:
-        c, e = do_test(options, inputs, type_word, names)
+    for type_word in ['status', 'algorithm', 'ecc_curve', 'dh_group',
+                      'key_type', 'key_usage']:
+        c, e = do_test(options, inputs, type_word)
```

Comment:
In commit "Move the type_word->name_set mapping into its own method", which says it has no change on behavior, we don't need to call `get_names()` anywhere? For example, while passing a names parameter to `do_test()`